starter-repos/shared-ci-cd/appctl/prepare.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+prepare-manifests:
+  only:
+    refs:
+    - master
+  stage: prepare-manifests
+  image: gcr.io/cloud-solutions-images/appctl
```

Comment:
@yuwenma Let me know if the appctl image URL is different

Reply:
kk 😄